src/documentation/programming-guide.md

```diff
@@ -1398,7 +1398,7 @@ If your `PCollection` is bounded (the size is fixed), you can assign all the ele
 %}
 ```
 
-### Time skew, data lag, and late data
+### <a name="watermarks-late-data"></a>Watermarks and late data
```

Comment:
Yeah, this could definitely be improved. I'm not familiar enough with CSS to do this offhand and don't want to block on it, but I'll add this to my list of things to take a look at. I tested moving it in to see how bad it looks if done without CSS for now, and well, it looks pretty bad!